src/test/java/com/ait/lienzo/test/LienzoCoreAttributesTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *    Copyright (c) 2018 Ahome' Innovation Technologies. All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+package com.ait.lienzo.test;
+
+import com.ait.lienzo.client.core.config.LienzoCore;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class LienzoCoreAttributesTest {
+
+    @Test
+    public void canEnableHiDPI()
```

Comment:
it seems this test does the same as the method below, right? Just keeping the one below is fine for me, no need to duplicate the assertions.

Reply:
Yes. The test bellow just alternate between true and false to be sure that the attribute can be set in the two states, because the original issue is that it can be set to any value and it always will have the default value (false). Am I overthinking the test?

Reply:
yep that's fine, makes sense, thanks ! :+1: